src/tree/SubscriptionTreeItem.ts

```diff
@@ -71,32 +93,46 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         return typeof contextValue !== 'string' || !/attached/i.test(contextValue);
     }
 
-    private async initChild(client: CosmosDBManagementClient, databaseAccount: DatabaseAccount): Promise<AzureTreeItem> {
-        const experience = tryGetExperience(databaseAccount);
-        const resourceGroup: string = azureUtils.getResourceGroupFromId(databaseAccount.id);
-        const accountKindLabel = getExperienceLabel(databaseAccount);
-        const label: string = databaseAccount.name + (accountKindLabel ? ` (${accountKindLabel})` : ``);
+    private async initChild(client: CosmosDBManagementClient | PostgreSQLManagementClient, databaseAccount?, databasesPostgres?): Promise<AzureTreeItem> {
```

Comment:
I would rename `initChild` to `initCosmosDBChild` and make this method exclusive to Cosmos. The postgres logic looks simple and unrelated